tools/vsimporter/xib2nib/src/UIButton.h

```diff
@@ -17,14 +17,34 @@
 #pragma once
 #include "UIControl.h"
 
+struct EdgeInsets {
```

Comment:
>EdgeInsets [](start = 7, length = 10)

We couldn't leverage the framework's UIEdgeInset? I guess this is the pain of the disconnect between our framework and this tool.